docs/stencil-docs/reference-docs/stencil-utils-api-reference.md

```diff
@@ -52,13 +51,34 @@ updateView() {
     this.refreshView(content);
   });
 }
- ```
+```
+
+The Cornerstone theme uses the `getPage` method in the following contexts:
 
-### Usage in Cornerstone
 - [`assets/js/theme/wishlist.js`](https://github.com/bigcommerce/cornerstone/blob/master/assets/js/theme/wishlist.js)
 - [`assets/js/theme/gift-certificate.js`](https://github.com/bigcommerce/cornerstone/blob/master/assets/js/theme/gift-certificate.js)
 - [`assets/js/theme/faceted-search.js`](https://github.com/bigcommerce/cornerstone/blob/master/assets/js/theme/common/faceted-search.js)
 
+### Get page by GraphQL
+
+[getPageByGQL](https://github.com/bigcommerce/stencil-utils/blob/master/src/api/index.js)
+
+This method takes the form `utils.api.getPageByGQL(page, callback)`. 
+
+It allows you to render parts of a template using an inline GraphQL query, as shown in [Front Matter Reference - GraphQL attributes](/stencil-docs/reference-docs/front-matter-reference#graphql-attributes).
+
+| Parameter | Type | Description/Usage |
+|:----------|:-----|:------------------|
+| page | string | Presentation template; for example, `pages/store-locator`. <br> The `template` option allows you to select a particular template, or an array of templates, to render one page. Each value must correspond to a file in the theme's `templates/pages/` subdirectory. |
+| callback | function | Asynchronous function call to handle the results. |
```

Comment:
defining a callback does not call it. revise this description & change throughout the doc.